src/Microsoft.Azure.IIoT.OpcUa/src/Registry/Models/SecurityEventModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
```

Comment:
Nitpick:  Could you move the namespaces inline for consistency with the other files and use the same formatting?